### ISSUE TITLE
Add hold-off before reporting high CPU load

### ISSUE THREAD

**Issue by mommsen** [cmsdaq/DAQExpert#186]
There have been several reports that the DAQExpert warns about a too high CPU load, e.g.:
```
HLT CPU load2018-05-07 08:40:41 23 s
HLT CPU load is high (( last: 92.9 %, avg: 96.6 %, min: 92.9 %, max: 99.6 %), which exceeds the threshold of 90.0%. Hide steps

    Call the HLT DOC, mentioning the HLT CPU load is high.

Called the HLT DOC -> short-term high load at beginning of run is expected
```

It is indeed expected that the CPU load is high at the start of the run during stable beams. I would suggest to add a hold-off of 1 minute before reporting the high CPU load.

Remi

**Comment by andreh12** [comment]
as discussed with @gladky , we will implement this by keeping track of the timestamp when the `RunOngoing` last went from `satisfied() = False` to `satisfied() = True` in the `HltCpuLoad` module.

**Comment by andreh12** [comment]
@mommsen is it ok to introduce a generic holdoff period since the beginning of the run or do you think we should apply the holdoff just for stable beams ?

**Comment by mommsen** [comment]
I think it is good enough to have a hold-off of 1 minute in any case. A short spike in CPU load does not cause any issues.

**Comment by andreh12** [comment]
added to release 2.10.6

**Comment by andreh12** [comment]
with @mommsen we agreed to use `expert.logic.hlt.cpu.load.holdoff.period = 180000` (3 minutes) in production

**Comment by gladky** [comment]
Reopening to include @hsakulin suggestion:

> Then we need two hold-offs:
> 
> E.g. 5 mins from the start of the run
> And 2 minutes from the start of the condition.

**Comment by gladky** [comment]
Fixed in 0914f77, released as 2.10.8
